projects/1.20/assets/ruins/philipsruins/lang/zh_cn.json

```
@@ -0,0 +1,40 @@
+{
+    "block.philipsruins.red_sand_stone_brick": "红砂岩砖",
+    "block.philipsruins.sand_stone_brick": "砂岩砖",
+    "block.philipsruins.red_sand_stone_tiles": "红砂岩瓷砖",
+    "block.philipsruins.obsidian_brick": "黑曜石砖",
+    "block.philipsruins.soul_sand_bricks": "灵魂沙砖",
+    "block.philipsruins.sandstone_tiles": "砂岩瓷砖",
+    "block.philipsruins.dark_brick": "暗色石砖",
+    "block.philipsruins.mossy_dark_brick": "苔藓暗色石砖",
+    "block.philipsruins.bone_bricks": "骨头砖",
+    "block.philipsruins.bone_bricks_slab": "骨头砖台阶",
+    "block.philipsruins.bone_chunk_block": "骨头块",
+    "block.philipsruins.chiseled_bone_bricks": "錾制骨头砖",
+    "block.philipsruins.cracked_bone": "裂纹骨头",
+    "block.philipsruins.fractured_bone_bricks": "破碎骨头砖",
+    "block.philipsruins.bone_bricks_stairs": "骨头砖楼梯",
+    "block.philipsruins.bone_bricks_wall": "骨头砖墙",
+    "block.philipsruins.sculk_bone_block": "菇骨块",
+    "block.philipsruins.sculk_bone_bricks": "菇骨砖",
+    "block.philipsruins.sculk_bone_bricks_slab": "菇骨砖台阶",
+    "block.philipsruins.sculk_bone_bricks_stairs": "菇骨砖楼梯",
+    "block.philipsruins.infested_sculk_bone_bricks": "被感染的菇骨砖",
+    "block.philipsruins.frozen_dark_prismarine": "冻结的深色海晶石",
+    "block.philipsruins.frozen_prismarine_bricks": "冻结的海晶石砖",
+    "block.philipsruins.frozen_sea_lantern": "冻结的海灯",

```

```suggestion
    "block.philipsruins.frozen_sea_lantern": "冻结的海晶灯",
```
参见原版`海晶灯`/`Sea Lantern`